src/server.ts

```diff
@@ -44,9 +45,9 @@ app.post('/getClinicalTrial', function (req, res) {
     const patientBundle = (typeof postBody.patientData === 'string' ? JSON.parse(postBody.patientData) : postBody.patientData) as Record<string, unknown>;
     if (isBundle(patientBundle)) {
       getResponse(patientBundle).then(result => {
-        console.log(result); // printing result from breastcancertrials.org for development
+        //console.log(result); // printing result from breastcancertrials.org for development
         console.log("END OF RESULTS");
-        const fhirResult = new SearchSet(result);
+        const fhirResult = new SearchSet(JSON.parse(JSON.stringify(result)) as TrialResponse[]);
```

Comment:
Why is this stringifying the result and then instantly un-stringifying it?

Reply:
I was getting type errors I couldn't resolve when just putting "result as TrialResponse[]" and it needs to be specified as that type to avoid other type errors during the build. I couldn't figure a more proper way to fix it, but if anyone knows a better way we should switch it

Reply:
**A** solution to force TypeScript to convert a type is to do something like `(result as unknown) as TrialResponse[]`. I need to look more closely at the code to figure out what the underlying error is.

Reply:
The type error was caused because `JSON` as a type in TypeScript ends up referring to the type of the `JSON` object within the global JavaScript namespace - that is, the object that contains the `parse` and `stringify` functions used in `JSON.parse` and `JSON.stringify`. It, unfortunately, does not refer to JSON objects. The type that TypeScript uses as the result from `JSON.parse` is, in fact, `any`, which of course our linter doesn't like. Generally the JSON type we want is something like `Record<string, unknown>` although the full type would be something more like `Record<string, unknown> | Array<unknown> | null | boolean | number | string` where `unknown` should really be the type being declared itself.